src/dex-ui-components/base/Icons/DefaultLogoIcon.tsx

```diff
@@ -0,0 +1,22 @@
+import { Icon } from "@chakra-ui/react";
```

Comment:
Use the `createIcon` approach for new Icons (the reasoning for this is in an early comment).

Reply:
Done, also same I updated for tooltipIcon